packages/appsync-modelgen-plugin/src/utils/process-belongs-to.ts

```diff
@@ -58,3 +53,35 @@ export function processBelongsToConnection(
     targetName: connectionFields[0] || makeConnectionAttributeName(model.name, field.name),
   };
 }
+
+export function getBelongsToConnectedField(field: CodeGenField, model: CodeGenModel, connectedModel: CodeGenModel, connectionInfo: CodeGenDirective): CodeGenField | undefined {
+  if(connectionInfo.arguments.fields) {
+    let indexDirective = flattenFieldDirectives(model).find(dir => {
+      return dir.name === 'index' && dir.fieldName === connectionInfo.arguments.fields[0];
+    });
+
+    if(indexDirective) {
+      let theIndex = indexDirective;
```

Comment:
The `indexDirective` can be used directly here instead of being reassigned to a new variable

Reply:
removed this code